build-system/tasks/extension-helpers.js

```diff
@@ -630,6 +630,18 @@ async function buildBentoCss(name, versions, minifiedAmpCss) {
 async function buildNpmBinaries(extDir, name, options) {
   let {npm} = options;
   if (npm === true) {
+    // remap all shared modules to the @bentoproject/core package and declare as external
+    // todo(kvchari): currently only used for bento standalone, investigate using for preact & react
+    const bentoEntryPoint = await getBentoBuildFilename(
+      extDir,
+      getBentoName(name),
+      'web-component',
+      options
+    );
+    const fullEntryPoint = path.join(extDir, bentoEntryPoint);
+    const remap = getRemapBentoNpmDependencies(fullEntryPoint);
+    const external = [...new Set(Object.values(remap))];
```

Comment:
Is this different because `remap` could contain dupe values?

Reply:
Yes. `remap` basically contains a map of X: Bento/Core, Y: Bento/Core, ..., so taking the values has a bunch of repeats